package.json

```diff
@@ -14,6 +14,8 @@
     "build:watch": "lerna run build:watch --parallel; tsc --build",
     "check": "yarn node ./tasks/check/check.mjs",
     "clean:prisma": "rimraf node_modules/.prisma/client && node node_modules/@prisma/client/scripts/postinstall.js",
+    "deps:dist": "cd ${INIT_CWD} && depcruise dist --config ${PROJECT_CWD}/.dependency-cruiser.mjs --output-type dot | dot -T svg > dependencyGraphDist.svg && echo \"Dependency graph written to ${INIT_CWD}/dependencyGraphDist.svg\"",
+    "deps:src": "cd ${INIT_CWD} && depcruise src --config ${PROJECT_CWD}/.dependency-cruiser.mjs --output-type dot --exclude \"^src/__tests__\" | dot -T svg > dependencyGraphSrc.svg && echo \"Dependency graph written to ${INIT_CWD}/dependencyGraphSrc.svg\"",
```

Comment:
Maybe these should just be calls to a script in tasks at this point given how long they are.